docs/draft/6-payloads.md

```diff
@@ -191,11 +209,28 @@ message ImageMessage {
 }
 ```
 
+##### Audio content type
+
+A `ChatMessage` with `AUDIO` `Content/Type` MUST also specify the `payload` of the audio
+and the `type`
+
+```protobuf
+message AudioMessage {
+  bytes payload = 1;
+  AudioType type = 2;
+  enum AudioType {
+    UNKNOWN_AUDIO_TYPE = 0;
+    AAC = 1;
+    AMR = 2;
+  }
+}
+```
+
 #### Message types
 
-A node requires message types to decide how to encrypt a particular message and what metadata needs to be attached when passing a message to the transport layer. For more on this, see [3/WHISPER-USAGE](./../stable/3-whisper-usage.md) and [10/WAKU-USAGE](./../stable/10-waku-usage.md).
+A node requires message types to decide how to encrypt a particular message and what metadata needs to be attached when passing a message to the transport layer. For more on this, see [3/WHISPER-USAGE](3-whisper-usage.md) and [10/WAKU-USAGE](10-waku-usage.md).
```

Comment:
what's going on with the remove of stable path?

Reply:
my bad, I have integrated the changes from `stable`